docs/Deprecate Automation.md

```diff
@@ -0,0 +1,85 @@
+# Automation Resource Decommission Instructions
```

Comment:
Are these instructions for the latest resources that are created with the Bicep or the previous resources created with PowerShell and Bash? If it is the older resources, we may not need this documentation since they won't be recreated in the future because the documentation has changed to use the new Bicep setup.

Reply:
@AAkindele , these instructions are for the previous resources created with PowerShell and Bash, I was thinking about do not have a separate documentation at all, and just put all these into the ticket or PR to be executed after the approval, however my goal was to have a master plan to execute the decommission resource task to do not miss any detail.  any suggestions?

Reply:
Makes sense. My recommendation would be to make a checklist of all the cleanup tasks in issue. That would allow you to keep track of the master plan without having to create a new doc that won't be relevant as soon as the cleanup is done.

Reply:
will do ,and request a new review.